end-to-end-test/local/specs/core/resultsviewDisabledTabs.spec.js

```diff
@@ -0,0 +1,47 @@
+var assert = require('assert');
+var goToUrlAndSetLocalStorageWithProperty = require('../../../shared/specUtils')
+    .goToUrlAndSetLocalStorageWithProperty;
+var { waitForOncoprint } = require('../../../shared/specUtils');
+var goToUrlAndSetLocalStorage = require('../../../shared/specUtils')
+    .goToUrlAndSetLocalStorage;
+
+const CBIOPORTAL_URL = process.env.CBIOPORTAL_URL.replace(/\/$/, '');
+const url = `${CBIOPORTAL_URL}/results/oncoprint?Action=Submit&RPPA_SCORE_THRESHOLD=2.0&Z_SCORE_THRESHOLD=2.0&cancer_study_list=study_es_0&case_set_id=study_es_0_all&data_priority=0&gene_list=ABLIM1%250ATMEM247&geneset_list=%20&genetic_profile_ids_PROFILE_COPY_NUMBER_ALTERATION=study_es_0_gistic&genetic_profile_ids_PROFILE_MUTATION_EXTENDED=study_es_0_mutations&profileFilter=0&tab_index=tab_visualize`;
+
+describe('results view check possibility to disable tabs', function() {
+    it('check that all tabs can be disabled', () => {
+        goToUrlAndSetLocalStorage(url, true);
+        waitForOncoprint();
+        $('.msk-tabs.posRelative.mainTabs').waitForDisplayed();
```

Comment:
not important but you could just use ".msk-tabs.mainTabs"   i.e. you don't have to list all classes

Reply:
1. I think waiting for both the Oncoprint and the tab container is redundant here. I think that for the purpouse of this test, waiting for the tab-container is sufficient. 

2. Also, overall it is good practice to make the code read like a story. This can be achieved for example like so:

```
const tabsContainer = $('.mainTabs');
tabsContainer.waitForDisplayed();
```

Please refactor all of the code in a similar fashion. The code must be self-explanatory.,

Reply:
Fixed.
The waitForOncoprint I added because some tabs are not ready yet and wasn't enough to just wait for the main tabs.